dotcom-rendering/src/model/enhance-dots.ts

```diff
@@ -2,26 +2,25 @@ import { transformDots } from './transformDots';
 
 const checkForDots = (elements: CAPIElement[]): CAPIElement[] => {
 	// Loop over elements and check if a dot is in the TextBlockElement
-	const enhanced: CAPIElement[] = [];
-	elements.map((element, i) => {
+	const enhanced: CAPIElement[] = elements.map((element, i) => {
```

Comment:
I think we should keep code changes outside of these kinds of lint PRs. I think for lots of reasons but perhaps mainly as we're going to exclude this commit from the history.

Reply:
I actually thing that if we’re changing a handful of elements, it’s okay to include these, because the diff truly is small. This whole PR is only `+56 -60`.

Reply:
Yes, as I was saying, I’ve moved to using `forEach` instead.

This is a typical example of doing something functional in a non-functional way and it puzzles me, but I agree that consistency is key. The ESLint rule is very helpful in pointing out us that using `map` without a return statement is **wrong**.